pages/publishing/publishing/adobe-experience-platform.mdx

```diff
@@ -0,0 +1,81 @@
+
+
+## Publishing to the Adobe Experience Platform
+
+To publish your Avo tracking plan to the Adobe Experience Platform, navigate to the AEP integration view in Avo and click the "Publish" button. Enable Auto Publishing to trigger an automatic publish whenever a branch has been merged.
+
+You can also download an XDM file, which you can import into AEP. 
+
+### What Is Included In The Publish to Adobe Experience Platform?
+
+On publish, Avo will fetch your existing XDM schemas, compare them with your Tracking Plan and publish new and updated XDM schemas. By default Avo will publish all events on the branch that you are on, but you can configure the publish to be [filtered by Sources, Destinations and Tags](/publishing/publishing/overview#filtering-events-for-publishing).
+
+> All entities published by Avo have `/avo/` in their ids, for example `https://ns.adobe.com/TENANT_ID/mixins/avo/my_properties`
+
+#### Events map to XDM Schemas
+
+The `Log Event` actions on Avo Events map to Adobe Experience Event schemas. 
+
+- **title**: Every included Avo Event with a `Log Event` action will create a XDM Experience Event schema with the same title as the event name. If [name mapping](/data-design/name-mapping) is applied to the event, the name defined by the name mapping is used here
+- **description**: This is the description as provided in Avo
+- **allOf**: Includes the `https://ns.adobe.com/xdm/context/experienceevent` and a field group, containing all the properties, attached to the `Log Event` action in Avo. See `Properties map to XDM Field Groups` below
+
+The `Update User Properties` actions on Avo Events map to Adobe Profile schemas.
+
+- **title**: Every included Avo Event with a `Update User Properties` action will create a XDM Profile schema with title `{Avo Event Name} Record`. If [name mapping](/data-design/name-mapping) is applied to the event, the name defined by the name mapping is used here
+- **description**: The event description as provided in Avo
+- **allOf**: Includes the `https://ns.adobe.com/xdm/context/profile` and a field group containing all the user properties attached to the `Update User Properties` action in Avo. See `Properties map to XDM Field Groups` below
+
+[Learn more about event actions in Avo](data-design/avo-tracking-plan/events#actions)
+
+
+#### Properties map to XDM Field Groups
+
+Experience Events:
+
+- **title**: All the properties of an included event with a `Log Event` action will create an XDM Field Group with title `{Avo Event Name} Properties`. If [name mapping](/data-design/name-mapping) is applied to the event, the name defined by the name mapping is used here
+- **definitions/property/properties/TENANT_ID/properties**: contains list of XDM properties created from the Avo event properties. `Type`, `title` with name mapping and `description` of each property is synced with Avo
```

Comment:
"`Type`, `title` with name mapping and `description` of each property is synced with Avo"

"Where `type` is the property type defined in Avo, `Title` is the property name or name mapping defined in Avo and `Descriotion` is the property description provided in Avo"